x-pack/plugin/ccr/src/main/java/org/elasticsearch/xpack/ccr/action/ShardFollowNodeTask.java

```diff
@@ -648,7 +679,14 @@ public long numberOfOperationsIndexed() {
             return fetchExceptions;
         }
 
+        private final long timeSinceLastFetchMillis;
```

Comment:
Thank you for keeping these in the same order as the constructor parameters, etc. 🙏